Pod/Classes/Optional Data Sources/VOKCollectionDataSource.m

```diff
@@ -270,19 +275,25 @@ - (void)controllerDidChangeContent:(NSFetchedResultsController *)controller
                             if (fetchLimit > 0) {
                                 if (indexPath.row < fetchLimit && newIndexPath.row < fetchLimit) {
                                     // Before and after are both in range
-                                    [self.collectionView moveItemAtIndexPath:indexPath toIndexPath:newIndexPath];
+                                    if (newIndexPath) {
+                                        [self.collectionView moveItemAtIndexPath:indexPath toIndexPath:newIndexPath];
+                                    }
```

Comment:
Yeah, you're right. I thought it would still need to execute the case on line 285, but that's not true after all.